opentreemap/importer/fields.py

```diff
@@ -3,29 +3,67 @@
 from __future__ import unicode_literals
 from __future__ import division
 
-
-class species(object):
+from treemap.models import Species
+
+
+class FieldNames(object):
+    @classmethod
+    def downcase_names(cls):
+        # Much code expects lower-case field names, but we want to show
+        # capitalized field names in the UI and in exported CSV's.
+        # So downcase everything but create a dict to map back
+        # (from a lowercased field name to its capitalized original).
+        def is_string(t):
+            return t is str or t is unicode
+
+        for attr in cls.__dict__.keys():
+            if attr[:1] != '_' and not '__' in attr:
+                val = getattr(cls, attr)
+                t = type(val)
+                if is_string(t):
+                    cls._capitalized_field_names[val.lower()] = val
+                    val = val.lower()
+                    setattr(cls, attr, val)
+                elif t is list or t is tuple or t is set:
+                    if len(val) > 0 and is_string(type(next(iter(val)))):
+                        # val is a sequence of strings
+                        val = t(item.lower() for item in val)
+                        setattr(cls, attr, val)
```

Comment:
If this code doesn't get replaced by, I think that this needs some heavy commenting.